solutions/landing-zone-v2/namespaces/monitoring.yaml

```diff
@@ -0,0 +1,82 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+# GCP SA
+# AC-3(7) - Creates lower priv service account for Policy Controller metrics
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMServiceAccount
+metadata:
+  name: metrics-sa
+  namespace: config-control # kpt-set: ${management-namespace}
+  annotations:
+    cnrm.cloud.google.com/project-id: management-project-id # kpt-set: ${management-project-id}
+    cnrm.cloud.google.com/ignore-clusterless: "true"
+spec:
+  displayName: metrics-sa
+---
+# Grant GCP role Metrics Writer to Metrics SA on KCC Project
+# AC-3(7) - RBAC role to account with required permissions for Policy Controller metrics
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMPolicyMember
+metadata:
+  name: metrics-writer-permissions
+  namespace: config-control # kpt-set: ${management-namespace}
+  annotations:
+    cnrm.cloud.google.com/project-id: management-project-id # kpt-set: ${management-project-id}
+    cnrm.cloud.google.com/ignore-clusterless: "true"
+spec:
+  resourceRef:
+    apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
+    kind: Project
+    external: management-project-id # kpt-set: ${management-project-id}
+  role: roles/monitoring.metricWriter
+  member: "serviceAccount:metrics-sa@management-project-id.iam.gserviceaccount.com" # kpt-set: serviceAccount:metrics-sa@${management-project-id}.iam.gserviceaccount.com
+---
+# K8S SA
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMPartialPolicy
+metadata:
+  name: metrics-sa-workload-identity-binding
+  namespace: config-control # kpt-set: ${management-namespace}
+  annotations:
+    cnrm.cloud.google.com/project-id: management-project-id # kpt-set: ${management-project-id}
+    cnrm.cloud.google.com/ignore-clusterless: "true"
+spec:
+  resourceRef:
+    name: metrics-sa
+    apiVersion: iam.cnrm.cloud.google.com/v1beta1
+    kind: IAMServiceAccount
+  bindings:
+    - role: roles/iam.workloadIdentityUser
+      members:
+        - member: serviceAccount:management-project-id.svc.id.goog[gatekeeper-system/gatekeeper-admin] # kpt-set: serviceAccount:${management-project-id}.svc.id.goog[gatekeeper-system/gatekeeper-admin]
+---
+# K8S namespace
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: monitoring
+  annotations:
+    cnrm.cloud.google.com/ignore-clusterless: "true"
+---
+# Link GCP SA to K8S namespace
+apiVersion: core.cnrm.cloud.google.com/v1beta1
+kind: ConfigConnectorContext
+metadata:
+  name: configconnectorcontext.core.cnrm.cloud.google.com
+  namespace: monitoring
+  annotations:
+    cnrm.cloud.google.com/ignore-clusterless: "true"
+spec:
+  googleServiceAccount: metrics-sa@management-project-id.iam.gserviceaccount.com # kpt-set: metrics-sa@${management-project-id}.iam.gserviceaccount.com
```

Comment:
For SA, I saw you use metrics-sa and namespace is monitoring .

Reply:
I created this sa specifically for Policy Controller GKE metrics. We could create a different namespace for gke-metrics perhaps or store everything monitoring-related inside this namespace.

Reply:
Fixed. I will resolve comment since it's been addressed already.